tests/test_audio_data.py

```diff
@@ -67,15 +67,39 @@ def test_cache_segment(random_item):
         assert torchaudio.load(f)
 
 def test_can_open():
-    p = Path('data/Right_whale.wav')
-    AudioItem.open(p)
-
+    p = Path('data/misc/whale/Right_whale.wav')
+    output = open_audio(p)
+    assert isinstance(output, AudioItem)
 
 def test_can_from_folder():
```

Comment:
Add a test for from_csv if you could as well

Reply:
This is still open. Didn't have a chance to add it tonight although I believe it will work now

Reply:
I have several personal projects outside the repo that are dependent on from_csv. I'll see if they work with this version and once we are aligned with fastai_V2 we can add proper testing with pytest.

Reply:
Great let us know when you've confirmed